modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/IgnitePdsWithTtlTest.java

```diff
@@ -34,43 +41,88 @@
+import org.apache.ignite.internal.processors.cache.persistence.GridCacheDatabaseSharedManager;
 import org.apache.ignite.internal.util.lang.GridAbsPredicate;
 import org.apache.ignite.internal.util.lang.GridCursor;
+import org.apache.ignite.internal.util.typedef.F;
 import org.apache.ignite.internal.util.typedef.PA;
+import org.apache.ignite.internal.util.typedef.X;
 import org.apache.ignite.internal.util.typedef.internal.CU;
+import org.apache.ignite.internal.util.typedef.internal.U;
 import org.apache.ignite.testframework.GridTestUtils;
 import org.apache.ignite.testframework.MvccFeatureChecker;
 import org.apache.ignite.testframework.junits.WithSystemProperty;
 import org.apache.ignite.testframework.junits.common.GridCommonAbstractTest;
 import org.junit.Test;
 
+import static org.apache.ignite.cache.CacheAtomicityMode.ATOMIC;
+import static org.apache.ignite.cache.CacheAtomicityMode.TRANSACTIONAL;
+import static org.apache.ignite.cluster.ClusterState.ACTIVE;
+import static org.apache.ignite.cluster.ClusterState.INACTIVE;
+import static org.apache.ignite.testframework.GridTestUtils.runAsync;
+import static org.apache.ignite.testframework.GridTestUtils.runMultiThreadedAsync;
+import static org.apache.ignite.testframework.GridTestUtils.waitForAllFutures;
+
 /**
  * Test TTL worker with persistence enabled
  */
 @WithSystemProperty(key = IgniteSystemProperties.IGNITE_UNWIND_THROTTLING_TIMEOUT, value = "5")
 public class IgnitePdsWithTtlTest extends GridCommonAbstractTest {
     /** */
-    public static final String CACHE_NAME = "expirableCache";
+    private static final String CACHE_NAME_ATOMIC = "expirable-cache-atomic";
+
+    /** */
+    private static final String CACHE_NAME_ATOMIC_NON_PERSISTENT = "expirable-non-persistent-cache-atomic";
+
+    /** */
+    private static final String CACHE_NAME_TX = "expirable-cache-tx";
+
+    /** */
+    private static final String CACHE_NAME_LOCAL_ATOMIC = "expirable-cache-local-atomic";
+
+    /** */
+    private static final String CACHE_NAME_LOCAL_TX = "expirable-cache-local-tx";
+
+    /** */
+    private static final String CACHE_NAME_NEAR_ATOMIC = "expirable-cache-near-atomic";
+
+    /** */
+    private static final String CACHE_NAME_NEAR_TX = "expirable-cache-near-tx";
 
     /** */
-    public static final String GROUP_NAME = "group1";
+    private static final String NON_PERSISTENT_DATA_REGION = "non-persistent-region";
 
     /** */
-    public static final int PART_SIZE = 32;
+    public static final int PART_SIZE = 2;
 
     /** */
     private static final int EXPIRATION_TIMEOUT = 10;
 
     /** */
     public static final int ENTRIES = 50_000;
 
+    /** */
+    public static final int SMALL_ENTRIES = 10;
+
+    /** */
+    private static final int WORKLOAD_THREADS_CNT = 16;
+
+    /** Fail. */
+    private volatile boolean fail;
+
     /** {@inheritDoc} */
     @Override protected void beforeTestsStarted() throws Exception {
```

Comment:
`beforeTestsStarted` and `afterTestsStopped` methods can be removed

Reply:
Fixed. Thank you!